pkg/cluster/cache.go

```diff
@@ -81,23 +79,21 @@ var Ready Condition = func(cluster *CachedToolchainCluster) bool {
 	return IsReady(cluster.ClusterStatus)
 }
 
-func (c *toolchainClusterClients) getCachedToolchainClustersByType(clusterType Type, conditions ...Condition) []*CachedToolchainCluster {
+func (c *toolchainClusterClients) getCachedToolchainClustersByType(conditions ...Condition) []*CachedToolchainCluster {
```

Comment:
The function name doesn't make much sense now. It's still called `getCachedToolchainClusterByType()` but there is no type anymore.